src/model.py

```diff
@@ -18,21 +22,38 @@ def __init__(
             self.num_patches,
             self.latent_space_dim
         )
+        self.transformer_blocks = nn.Sequential(
+            *[TransformerBlock(latent_space_dim, dim_ff, num_heads, self.num_patches) for _ in range(depth)]
+        )
+        self.classification_head = FeedForward(self.num_patches * latent_space_dim, dim_ff, num_classes)
```

Comment:
This is something I am unsure about. I would expect we flatten the embeddings so that we have one embedding per picture and then try to predict the class from this embedding. But apparently, in the paper, they do some kindof averaging over all patches. Will have to look into this